worlds/tunic/__init__.py

```diff
@@ -245,17 +245,27 @@ def extend_hint_information(self, hint_data: Dict[int, Dict[int, str]]):
                     continue
                 path_to_loc = []
                 previous_name = "placeholder"
-                name, connection = paths[location.parent_region]
-                while connection != ("Menu", None):
-                    name, connection = connection
-                    # for LS entrances, we just want to give the portal name
-                    if "(LS)" in name:
-                        name, _ = name.split(" (LS) ")
-                    # was getting some cases like Library Grave -> Library Grave -> other place
-                    if name in portal_names and name != previous_name:
-                        previous_name = name
-                        path_to_loc.append(name)
-                hint_text = " -> ".join(reversed(path_to_loc))
+                try:
+                    name, connection = paths[location.parent_region]
+                except:
+                    # logic bug, proceed with warning since it takes a long time to update AP
+                    warning(f"{location.name} is not logically accessible for "
+                            f"{self.multiworld.get_file_safe_player_name(self.player)}. "
+                            "Creating entrance hint Inaccessible. "
+                            "Please report this to the Tunic rando devs.")
```

Comment:
This is not nearly funny enough, needs to have at least 3 layers of self-deprecation or flirty vibes /j

Reply:
So true